docs/getting-started/testing-solidus.mdx

```diff
@@ -135,7 +150,7 @@ module AmazingStore
 
         # Optional: Any factories you want to require from extensions.
         extension_paths = [
-          # MySolidusExtension::Engine.root.join("lib/solidus_content/factories/order.rb"),
+          # MySolidusExtension::Engine.root.join("lib/solidus_content/factories.rb"),
```

Comment:
Not a result of this PR but there is some inconsistent naming between the ruby namespace and the path, should be:

```rb
# MySolidusExtension::Engine.root.join("lib/my_solidus_extension/factories.rb"),
# MySolidusExtension::Engine.root.join("lib/my_solidus_extension/factories/product.rb"),
```